dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -134,38 +133,58 @@ public String getStatus() {
 	}
 
 	public ESIndexAPI(){
-		this.esclient = new ESClient();
 		this.iapi = new ContentletIndexAPIImpl();
-		this.esIndexHelper = ESIndexHelper.INSTANCE;
-		this.serverAPI = APILocator.getServerAPI();
+		this.esIndexHelper = ESIndexHelper.getInstance();
 		this.clusterAPI = APILocator.getClusterAPI();
 	}
 
-	@VisibleForTesting
-	protected ESIndexAPI(final ESClient esclient, final ContentletIndexAPIImpl iapi, final ESIndexHelper esIndexHelper,
-						 final ServerAPI serverAPI, final ClusterAPI clusterAPI){
-		this.esclient = esclient;
-		this.iapi = iapi;
-		this.esIndexHelper = esIndexHelper;
-		this.serverAPI = serverAPI;
-		this.clusterAPI = clusterAPI;
-	}
+    private class IndexSortByDate implements Comparator<String> {
+        public int compare(String o1, String o2) {
+            if (o1 == null || o2 == null) {
+                return 0;
+            }
+            if (o1.indexOf("_") < 0) {
+                return 1;
+            }
+            if (o2.indexOf("_") < 0) {
+                return -1;
+            }
+            String one = o1.split("_")[1];
+            String two = o2.split("_")[1];
+            return two.compareTo(one);
+        }
+    }
 
-    /**
-     * returns all indicies and status
-     * @return
-     */
-    public Map<String,IndexStats> getIndicesAndStatus() {
-        final Client client = esclient.getClient();
-        final IndicesStatsResponse
-            indicesStatsResponse =
-                client.admin().indices().prepareStats().setStore(true).execute().actionGet(INDEX_OPERATIONS_TIMEOUT_IN_MS);
+	@SuppressWarnings("unchecked")
+	public Map<String, IndexStats> getIndicesStats() {
+		final Request request = new Request("GET", "/_stats");
+		final Map<String, Object> jsonMap = performLowLevelRequest(request);
 
-        return indicesStatsResponse.getIndices();
-    }
+		Map<String, IndexStats> indexStatsMap = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'indexStatsMap' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)